src/test/integration/java/com/commercetools/pspadapter/paymentHandler/impl/PaymentHandlerProviderImplIT.java

```diff
@@ -201,12 +210,56 @@ public void shouldCreateChargeTransaction() {
         assertThat(chargeTxn.getState()).isEqualTo(SUCCESS);
     }
 
+    @Test
+    public void whenCtpPaymentIsAlreadyMarkedAsSuccess_shoudNotCallCreateTransactionInPaypalPlus() {
+
+        final CtpPaymentWithCart ctpPaymentWithCart = createCartWithPayment();
+        final PaymentHandler paymentHandler = paymentHandlerProvider
+                .getPaymentHandler(MAIN_TEST_TENANT_NAME)
+                .get();
+
+        io.sphere.sdk.payments.Payment payment = ctpPaymentWithCart.getPayment();
+
+        final TransactionDraft transactionDraft = TransactionDraftBuilder
+                .of(CHARGE, Money.of(200, EUR)).state(TransactionState.SUCCESS)
+                .build();
+        final AddTransaction addTransaction = AddTransaction
+                .of(transactionDraft);
+        final SetInterfaceId interfaceIdUpdateAction = SetInterfaceId.of("testInterfaceId");
+        List<UpdateAction<io.sphere.sdk.payments.Payment>> actions = new ArrayList<>();
+        actions.add(addTransaction);
+        actions.add(interfaceIdUpdateAction);
+
+        final io.sphere.sdk.payments.Payment updatedPayment = this
+                .sphereClient
+                .execute(PaymentUpdateCommand.of(payment, actions))
+                .toCompletableFuture()
+                .join();
+
+        final String ctpPaymentId = updatedPayment.getId();
+
+        executeBlocking(paymentHandler.createPayment(ctpPaymentId));
+
+        PaymentHandleResponse response = executeBlocking(paymentHandler
+                .executePayment(updatedPayment.getInterfaceId(), "testPayerId"));
+
+        io.sphere.sdk.payments.Payment ctpPayment = executeBlocking(sphereClient
+                .execute(PaymentByIdGet.of(ctpPaymentId)));
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(ctpPayment.getInterfaceInteractions()).hasSize(1);
+    }
+
+
     @Test
     public void whenPaypalPaymentIsNotApproved_transactionShouldNotBeCreated() {
-        CtpPaymentWithCart ctpPaymentWithCart = createCartWithPayment();
-        PaymentHandler paymentHandler = paymentHandlerProvider.getPaymentHandler(MAIN_TEST_TENANT_NAME).get();
 
```

Comment:
Why this blank line?

Reply:
de5f9ee05b93c102ced24ca1c8f961aaa951a64e